### ISSUE TITLE
Section 5.4 Third Party Content is missnamed

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#5263]
[Section 5.4](https://www.w3.org/TR/WCAG22/#conformance-partial) Statement of Partial Conformance — Third Party Content was the topic of scrutiny on a recent AG call.  See: [minutes](https://www.w3.org/2026/08/04-ag-minutes.html#3497) and [survey results](https://www.w3.org/wbs/35422/wcag3-document-breakdown/results/) (questions 7 and 8).

The term “third party content” has come to have much broader implications than how the term is used in WCAG 2.x.  The survey, for example, distinguishes between [3rd party vendor content (procured)](https://www.w3.org/wbs/35422/wcag3-document-breakdown/results/#x3rdparty) and [User-generated content](https://www.w3.org/wbs/35422/wcag3-document-breakdown/results/#xusergen).

In the context of WCAG 2.x, “3rd party vendor content (procured)” is just “content”.  Auditors frequently hear site owner protest that certain content is not under their control, but that’s a consequence of conformance being applicable for [full pages](https://www.w3.org/TR/WCAG22/#cc2).

I submit that the prose of Section 5.4 is fine but that “third party content” in the title is misleading.  The prose discusses content aggregation, “user-contributed content” and dynamic advertisements.

What these have in common is being “uncontrolled content” — and I propose that as a better term to appear in the title of this section.

But should user-generated content be addressed differently than aggregation and advertisements?  Can this issue be addressed as an erratum?

**Comment by mraccess77** [comment]
I think 3rd party content should be treated differently than user generated content.  Third party content to which you choose to use requires the author to be responsible for it's use.  

For user generated content - the responsibility is to support the author in being able to contribute accessible content - but not forcing them to provide accessible content.

**Comment by bruce-usab** [comment]
@mraccess77 do you think it a mistake that 5.4 currently includes the examples of content aggregation and advertising?

**Comment by bruce-usab** [comment]
Noting that Statement of Partial Conformance is the subject of [new WG survey](https://www.w3.org/wbs/35422/ugc_section_revisions/results/) and [draft Google Doc](https://docs.google.com/document/d/1oaKWQzC8UKgY16STXsbbdTvlhX4K89zM5DZlrJD99Xk/edit?tab=t.0#heading=h.8uuizr9ubuyr).

**Comment by mraccess77** [comment]
I think we have several separate things that need to be disambiguated.
* User generated content and the ability to make the content accessible
* Third party content under control, arranged for, contracted, etc.
* Third party content - not user generated and not under control but which is included in the page
* Third party content linked as informational only and not needed to perform the task

It seems like ads could fall into ones that are controlled and not controlled - while the person may not have control over one of them they still impact the user and they are not user generated.
